### ISSUE TITLE
XeTeX math font does not show on Ipe display

### ISSUE THREAD

**Issue by GoBigorGoHome** [otfried/ipe#314]
# System Info:
Windows  10.0.19041.208
TeX Live 2020 (the bin folder added to PATH)
Ipe version 7.2.17 (there is the same problem in version 7.2.16)

# How to reproduce:
Go to Edit -> Document properties, set LaTeX engine to xetex or luatex.
Add a text label with some math formula like `$a+b$`.
With xetex, the result text label is empty; with luatex, an error message pops up.

**Comment by otfried** [comment]
Today I managed to install Texlive 2020.   It turns out the Luatex problem is very simple, and I'll fix it in the next release.

I cannot see any problem with the xetex engine.   If you have an Ipe file that doesn't work with xetex, can you attach it here?

**Comment by GoBigorGoHome** [comment]
[sample.zip](https://github.com/otfried/ipe-issues/files/4634016/sample.zip)

As shown in the screen shot below, with xetex, math formula is not rendered while text is rendered fine.

![sample](https://user-images.githubusercontent.com/9896288/82045458-b5324500-96e1-11ea-9060-dc4b681a1bd5.png)

xetex version info on my machine:
```txt
C:\Users\***>xetex --version
XeTeX 3.14159265-2.6-0.999992 (TeX Live 2020/W32TeX)
kpathsea version 6.3.2
Copyright 2020 SIL International, Jonathan Kew and Khaled Hosny.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the XeTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the XeTeX source.
Primary author of XeTeX: Jonathan Kew.
Compiled with ICU version 65.1; using 65.1
Compiled with zlib version 1.2.11; using 1.2.11
Compiled with FreeType2 version 2.10.1; using 2.10.1
Compiled with Graphite2 version 1.3.13; using 1.3.13
Compiled with HarfBuzz version 2.6.4; using 2.6.4
Compiled with libpng version 1.6.37; using 1.6.37
Compiled with poppler version 0.68.0
Compiled with fontconfig version 2.13.92; using 2.13.92
```

**Comment by otfried** [comment]
This is very mysterious.   I have exactly the same version of xetex, and for me the letter displays correctly using the released version of Ipe 7.2.17.

Could you open this sample file again in Ipe, press Ctrl+L to run Latex, and then upload the files ipetemp.tex, ipetemp.log, and ipetemp.pdf from the Ipe Latex directory (you can find that in Help -> Show configuration, it is typically C:\Users\name\AppData\Local\ipe) ?

**Comment by GoBigorGoHome** [comment]
[log.zip](https://github.com/otfried/ipe-issues/files/4640636/log.zip)

ipetemp.pdf looks good, but in ipe 'a' is still missing.

**Comment by otfried** [comment]
If you save the Ipe document in PDF format, does the letter "a" show up correctly?

**Comment by GoBigorGoHome** [comment]
"a" shows up correctly in the pdf file. However, opening that same pdf in Ipe and "a" is still missing in the Ipe window.

![sample_pdf](https://user-images.githubusercontent.com/9896288/82155909-ab0f7280-98aa-11ea-83ab-4ad9d149bdf4.png)

**Comment by otfried** [comment]
That's interesting!  Can you attach sample.pdf?

**Comment by GoBigorGoHome** [comment]
[sample.pdf](https://github.com/otfried/ipe-issues/files/4640734/sample.pdf)

**Comment by otfried** [comment]
This is so weird, I just cannot reproduce it.

If you open sample.pdf in ipepresenter, does the "a" show up?

**Comment by GoBigorGoHome** [comment]
No.
Does Ipe have a debug mode? Maybe something is wrong with my Ipe or texlive installation or even my operating system configuration.

**Comment by otfried** [comment]
Can you show a screen shot of ipepresenter (not Ipe) with sample.pdf?

Ipepresenter is quite self-contained, it renders the PDF file directly using bundled libraries.  So if the letter does not show up, it cannot be because of texlive, and it's hard to see how the OS can cause this.

**Comment by GoBigorGoHome** [comment]
![pdf_in_ipepresenter](https://user-images.githubusercontent.com/9896288/82157757-fa0ed500-98b5-11ea-9b82-aa511183f661.png)

**Comment by otfried** [comment]
And you are using the most recent version of `ipe-7.2.17-win64.zip` ?

**Comment by otfried** [comment]
And this problem only started after you upgraded to Texlive 2020?

**Comment by GoBigorGoHome** [comment]
> And you are using the most recent version of ipe-7.2.17-win64.zip ?

Yes.

> And this problem only started after you upgraded to Texlive 2020?

Just switched to texlive 2019 (by modifying PATH), there is the same issue with xetex.

**Comment by otfried** [comment]
I'm afraid I have no more ideas as to what to try for the moment.

Ipepresenter is completely self-contained - it doesn't need Latex, and it uses its bundled libraries to display the PDF file.   So I don't understand why it displays `sample.pdf` correctly on all my computers but not on yours.

If you put Ipe and `sample.pdf` on a USB-stick, you can try opening `sample.pdf` with ipepresenter on another computer and see if it works there.

**Comment by otfried** [comment]
Could you try a few different text objects?

You had one with contents $a$ + b

What happens if it is just $a$  ?

What happens if it is a + $b$  ?

This is a really mysterious problem.

**Comment by GoBigorGoHome** [comment]
I have no acccess to another computer at the moment.

The problem is the same with $a$ or a + $b$.

**Comment by otfried** [comment]
So in both cases the math font does not show up, but the normal font does?

**Comment by GoBigorGoHome** [comment]
Yes.

**Comment by GoBigorGoHome** [comment]
The issue disappear now for Ipe 7.2.19.

**Comment by pengdlzn** [comment]
I had the same problem, where the numbers in math format did not show up as if they were transparent. I am using Ipe 7.2.24.1 and MikTex 21.6 on Windows 10. I use Latex engine "xetex" and package "xeCJK" because I may write Chinese in the images. As I do not have Chinese text yet, I removed "\usepackage{xeCJK}" in "Document properties", and the numbers in math format show up correctly. Then, I added "\usepackage{xeCJK}" back, and the numbers still show up correctly.
